content/en/docs/community/release_checklist.md

```diff
@@ -126,31 +158,34 @@ git tag --sign --annotate "$RELEASE_NAME" --message "Helm release $RELEASE_NAME"
 git push upstream "$RELEASE_NAME"
 ```
 
-This new tag is going to be the base for the patch release.
+This new tag is going to be the base for the patch release. Creating a
+[helm/helm milestone](https://github.com/helm/helm/milestones) is optional for
+patch releases.
 
 Make sure to check [helm on CircleCI](https://circleci.com/gh/helm/helm) to see
-that the release passed CI before proceeding.
+that the release passed CI before proceeding. Patch releases can skip steps 2-6
+and proceed to step 7 for [PGP signature](#7-pgp-sign-the-downloads).
 
-## 2. Change the Version Number in Git
+## 2. Major/Minor releases: Change the Version Number in Git
 
-When doing a minor release, make sure to update pkg/version/version.go with the
-new release version.
+When doing a major or minor release, make sure to update
+`internal/version/version.go` with the new release version.
 
 ```shell
-$ git diff pkg/version/version.go
-diff --git a/pkg/version/version.go b/pkg/version/version.go
-index 2109a0a..6f5a1a4 100644
---- a/pkg/version/version.go
-+++ b/pkg/version/version.go
-@@ -26,7 +26,7 @@ var (
+$ git diff internal/version/version.go
+diff --git a/internal/version/version.go b/internal/version/version.go
+index 712aae64..c1ed191e 100644
+--- a/internal/version/version.go
++++ b/internal/version/version.go
+@@ -30,7 +30,7 @@ var (
         // Increment major number for new feature additions and behavioral changes.
         // Increment minor number for bug fixes and performance enhancements.
         // Increment patch number for critical fixes to existing releases.
--       Version = "v2.6"
-+       Version = "v2.7"
+-       version = "v3.3"
++       version = "v3.4"
 
-        // BuildMetadata is extra build time data
-        BuildMetadata = "unreleased"
+        // metadata is extra build time data
+        metadata = ""
```

Comment:
This change is out of scope for the current set of updates but I'm going with what's current now.